docs/src/userguide/wrapping_CPlusPlus.rst

```diff
@@ -554,14 +554,16 @@ Specify C++ language in setup.py
 Instead of specifying the language and the sources in the source files, it is
 possible to declare them in the :file:`setup.py` file::
 
-   from setuptools import setup
-   from Cython.Build import cythonize
+    from distutils.core import setup
```

Comment:
Is there a good reason to change this back to distutils? it looks like it was set to setuptools deliberately https://github.com/cython/cython/pull/3456

Reply:
Yup my bad, I dont this that is necessary.
However, switching back to setuptools, I'm getting a weird compiling error, i think it's due to my MacOS
Will provided more information ASAP